azext_edge/edge/providers/orchestration/work.py

```diff
@@ -368,6 +378,10 @@ def _do_work(self):
                     verify_write_permission_against_rg(
                         subscription_id=self.subscription_id, resource_group_name=self._targets.resource_group_name
                     )
+                if self._check_cluster:
+                    load_config_context(context_name=self._context_name)
```

Comment:
I think `load_config_context()` should not happen in the work module.

Reply:
while we're on the subject of what goes where, regarding the [comment ](https://github.com/Azure/azure-iot-ops-cli-extension/pull/546#discussion_r2071889034) about storage class validation - does that make more sense to bring up in the `cluster_kwarg` building, or should we have that validation fire in the `_check_storage_classes` function?